stablehlo/dialect/BroadcastUtils.cpp

```diff
@@ -22,12 +22,16 @@ limitations under the License.
 #include "llvm/ADT/Sequence.h"
 #include "llvm/ADT/SmallVector.h"
 #include "mlir/Dialect/Shape/IR/Shape.h"
+#include "stablehlo/dialect/Base.h"
 
 namespace mlir {
 namespace hlo {
 
 bool isLegalNumpyRankedBroadcast(Value lhs, Value rhs,
-                                 DenseIntElementsAttr broadcastDims) {
+                                 Attribute broadcastDimensionsAttr) {
```

Comment:
Can we make this an ArrayRef and lift the call to `getI64Array` to the callsite?